src/test/kotlin/uk/gov/dluhc/printapi/testsupport/testdata/entity/CertificateBuilder.kt

```diff
@@ -143,12 +143,12 @@ fun buildElectoralRegistrationOffice(
 }
 
 fun buildAddress(
-    street: String = DataFaker.faker.address().streetName(),
-    postcode: String = DataFaker.faker.address().postcode(),
-    property: String? = DataFaker.faker.address().buildingNumber(),
-    locality: String? = DataFaker.faker.address().streetName(),
-    town: String? = DataFaker.faker.address().city(),
-    area: String? = DataFaker.faker.address().state(),
+    street: String = faker.address().streetName(),
+    postcode: String = faker.address().postcode(),
```

Comment:
minor (non-blocking) I know you didn't write as part of this pr but this creates a new fake address object, takes one attribute, creates a whole new fake address object, takes one attribute, ...
I think when creating fake addresses we should do something like this 
```
    fakeAddress: net.datafaker.providers.base.Address = faker.address(),
    street: String = fakeAddress.streetName(),
    postcode: String = fakeAddress.postcode(),
```